File-Formats/PDF/Split-Documents.md

```diff
@@ -292,4 +292,77 @@ End Using
 
 {% endtabs %}
 
-Download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Split%20PDFs/Split-PDF-based-Bookmarks/.NET).
\ No newline at end of file
+Download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Split%20PDFs/Split-PDF-based-Bookmarks/.NET).
+
+## Remove Unused Resources when Splitting PDF Documents
+
+The Syncfusion PDF library enables the splitting of PDF documents and offers the capability to eliminate unused resources during the process. By enabling the **RemoveUnusedResources** property on the **PdfSplitOptions** class, any resources that are not in use will be deleted, thereby optimizing the final PDF document. The default value for this property is false.
```

Comment:
Can we use the highlight instead of adding bold text like.

The Syncfusion PDF library enables the splitting of PDF documents and offers the capability to eliminate unused resources during the process. By enabling the `RemoveUnusedResources` property on the `PdfSplitOptions` class, any resources that are not in use will be deleted, thereby optimizing the final PDF document. The default value for this property is false.

Reply:
modified